emailApi/email.js

```diff
@@ -0,0 +1,27 @@
+
+exports.mailSend = (eFrom,eTo,eSubject,eText)=>{
```

Comment:
Accept json object as a parameter instead of receiving eFrom, eTo .. separate params